smac/optimizer/smbo.py

```diff
@@ -130,7 +126,7 @@ def __init__(self,
 
         self.initial_design_configs = []  # type: typing.List[Configuration]
 
-        # initialize the chooser to get configurations from the EPM
+        # TODO consider if we need an additional EPMChooser for multi-objective optimization
```

Comment:
do we need one?

Reply:
Probably. Because we would need an EPM for each objective.